src/App.css

```diff
@@ -1,6 +1,10 @@
+* {
```

Comment:
I'd add `box-sizing: border-box` 

you can take a look why this is helpful here:
https://css-tricks.com/box-sizing/

Reply:
I added a box-sizing: border box.